1.preprocess_data/scripts/1.extract_metadata.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pathlib
+
+import pandas as pd
+
+# In[2]:
+
+
+path_to_data = pathlib.Path("../../data/images").resolve(strict=True)
+
+
+# In[3]:
+
+
+# get all files in the data directory
+files = path_to_data.glob("*")
```

Comment:
Will it always be the case that no other files exist here? Perhaps adding a `*.tiff` would future proof if so

also, `files` is a non-specific name - try to be more descriptive in variable naming.

Reply:
renamed variable and added specific glob parsing